fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -530,6 +530,35 @@ struct DDQueue : public IDDRelocationQueue {
 		Future<Void> cancel;
 	};
 
+	struct MovingAverage {
```

Comment:
In the future we might consider moving this to a more generic class as this might be useful for logging things beyond just this metric. (I don't think we need to do that in this PR.)

Reply:
I am not sure where I should move the struct, maybe `DataDistribution.actor.h` or `DDRelocationQueue.h`?